google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/integration/AppEngineServerIntegration.java

```diff
@@ -48,4 +55,9 @@ public ApplicationServerPersistentDataEditor createNewServerEditor() {
     // project wizard because there is a special 'Google App Engine' option
     return null;
   }
+
+  @Override
+  public ApplicationServerHelper getApplicationServerHelper() {
```

Comment:
See the version of this class in master - this way you can get rid of `AppEngineServerHelper` completely.